### DISCUSSION TITLE
List Identity Store IDs

### DISCUSSION THREAD

**Discussion by Kalpa2019** [aws/aws-sdk-java-v2#3447] [Q&A]
Hi All,
     Identitystore  has  method like listUser/groupetc. we need to pass identity store id for that,  is there any way that we can get the list of identity store ids programmatically and query the rest of the APIs.

thanks

**Comment by yasminetalby** [comment]
Hello @Kalpa2019 ,

Thank you very much for your submission
What would be the use case of this? 

Best,

Yasmine
